server/views/pages/overview/partials/overviewTab/overviewTabContents.njk

```diff
@@ -8,6 +8,17 @@
   </div>
 
   <div class="govuk-grid-column-one-third app-u-print-full-width">
+    {{ actionsCard({
+      actions: [
+        { title: 'Add a new goal', href: '/plan/' + prisonNumber + '/goals/create', id: 'add-goal-button', 'render-if': hasEditAuthority },
+        { title: 'View archived goals', href: '#' }
```

Comment:
We'll come back and update the href for the "View archived goals" action item once we have created that route etc